src/gengowatcher/ui_textual.py

```
@@ -3124,3 +3118,21 @@ def _colorize_message(self, msg: str, level: int) -> Text:
         return text
+
+
+
+# ==============================================================================
+# Backward-compatibility alias (legacy test contract).
+# This shim preserves the pre-refactor public surface so older tests keep
+# working until they are migrated to the new (render_chart_with_axes) impl.
+# ==============================================================================
+try:
+    from .ui_charts import render_chart_with_axes as _render_chart  # type: ignore[attr-defined]
+except Exception:  # pragma: no cover - charts module not always available
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)